andela/metis-ah-frontend
# #159987354 Setup an empty react project

**What does this PR do?**
Setup up an empty react project

**Description of Task to be completed?**

- setup Webpack for module bundling
- configure ESlint
- render build from server side with `NodeJS`

**How should this be manually tested?**

Clone the **repository**

Check out the `ch-setup-react-project-#159987354` branch
Run `npm install` on your local machine to install required dependencies

Run  `npm run dev`

**What are the relevant pivotal tracker stories?**
#159987354
